test/js/test.js

```diff
@@ -125,6 +126,7 @@ var state = {
   authorizationURL: "",
   responseURL: "",
   path: "",
+  abortStep: cliOptions.abortStep,
```

Comment:
No need to make this part of the state object, just reference cliOptions.abortStep directly below.